cmake/treedata/optional/common/gbm.txt

```diff
@@ -1,2 +1,3 @@
 xbmc/cores/RetroPlayer/process/gbm cores/RetroPlayer/process/gbm # GBM
-xbmc/windowing/gbm windowing/gbm # GBM
+xbmc/cores/VideoPlayer/Process/gbm cores/VideoPlayer/Process/gbm # GBM
+xbmc/windowing/gbm windowing/gbm # GBM
```

Comment:
don't remove the empty line at the end of file

Reply:
thanks, fixed